src/cli/probe.ts

```diff
@@ -0,0 +1,64 @@
+import { inputData, outputData } from '../lib/io';
+import { invokeUserFn, loadUserFn } from './lib/user-fn';
+import { verifyInputArray } from '../lib/verify';
+import {
+  standardCmdInputs,
+  standardCmdOutputs,
+  standardInputFileHelp,
+  standardOutputFileHelp,
+} from './lib/help';
+
+export async function main(argv: string[]): Promise<void> {
+  const { fn, details } = loadUserFn(argv, `record => transform(record)`);
+
+  const data = await inputData(argv);
+  verifyInputArray(data, 'map');
```

Comment:
This should take the name of the command to display for the help, which is 'probe' instead of 'map'.